app/models/guest.rb

```diff
@@ -0,0 +1,6 @@
+class Guest < ApplicationRecord 
+  has_many :guest_rooms 
+  has_many :rooms, through: :guest_rooms
+  validates_presence_of :name
+  validates_numericality_of :nights
```

Comment:
Nice job validating your data! Just so you are aware, we won't ask you to do this for the final or other ICs, just projects.